pkg/stream/enviroment.go

```diff
@@ -104,7 +104,11 @@ func (env *Environment) DeclareStream(streamName string, options *StreamOptions)
 	if err != nil {
 		return err
 	}
-	return client.DeclareStream(streamName, options)
+	errCreateStream := client.DeclareStream(streamName, options)
+	if errCreateStream == StreamAlreadyExists {
+		return nil
+	}
+	return errCreateStream
```

Comment:
**nit:** could call the error just `err` since there are no other operations here. Could also making it inline on the if like
```
if err := client.DeclareStream(...); err != nil && err != StreamAlreadyExists {
    return err
}
return nil
```